case-studies/openai/chatgpt/chatGPT-case-study-oskar-heming.jenssen.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ChatGPT Case Study
+author: Oskar Heming Jenssen - ohjenssen
+tags: chatgpt, case study, platform, developement-platforms, openai, machine-learning, artificial-intelligence
+---
+
+## Introduction
+
+ChatGTP is an artifical intelligence chatbot developed by the research laboratory OpenAI and launched as a prototype in november 30, 2022. ChatGPT is part of the Generative Pre-trainer Transformer (GPT) family of language models, which refers to how ChatGPT processes requests and formulates responses.
```

Comment:
```suggestion
ChatGPT is an artificial intelligence chatbot developed by the research laboratory OpenAI and launched as a prototype on 30 November 2022. ChatGPT is part of the Generative Pre-trained Transformer (GPT) family of language models, which refers to how ChatGPT processes requests and formulates responses.
```

Reply:
Please proof read and spell check. I recommend using Grammarly, Code Spell Checker and GPT to do this.